backend/main.py

```
@@ -3,465 +3,483 @@
+            agent_service = initialize_agent_service(self.orchestrator)
+            set_mcp_filesystem_service(mcp_service)
+            service_registry.register(agent_service)
+            logger.info("✅ Agent service initialized")
+
             self._services_initialized = True
             logger.info("🎉 All services initialized successfully!")
-            
+
         except Exception as e:
             logger.error(f"❌ Service initialization failed: {e}")
             raise SwarmError(f"Service initialization failed: {str(e)}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)